lib/system/excpt.nim

```diff
@@ -443,12 +463,15 @@ proc raiseExceptionEx(e: sink(ref Exception), ename, procname, filename: cstring
                       line: int) {.compilerRtl, nodestroy.} =
   if e.name.isNil: e.name = ename
   when hasSomeStackTrace:
-    if e.trace.len == 0:
-      rawWriteStackTrace(e.trace)
-    elif framePtr != nil:
-      e.trace.add reraisedFrom(reraisedFromBegin)
-      auxWriteStackTrace(framePtr, e.trace)
-      e.trace.add reraisedFrom(reraisedFromEnd)
+    when defined(nimStackTraceOverride):
+      e.trace = @[]
```

Comment:
Er ... what? Can't this be supported too?

Reply:
Here's what happened before:
- `rawWriteStackTrace(e.trace)` was called, because the length was 0
- since NimStackTrace is not defined, e.trace is set to @[]: https://github.com/nim-lang/Nim/blob/e1d79d40f2ef41e26b6c529f52023a59acaf0d0e/lib/system/excpt.nim#L304-L308
So I kept the logic and avoided a function call..

Now, I don't see `e.trace` being anything other than an empty seq when I print it with `--stacktrace:off`. It gets filled with `--stacktrace:on` but, since the whole point is to replace that, I want to empty that seq, which is why the "nimStackTraceOverride" case is the first one checked.